test/cpp_extensions/extension.cpp

```diff
@@ -37,4 +41,17 @@ PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
       .def(py::init<int, int>())
       .def("forward", &MatrixMultiplier::forward)
       .def("get", &MatrixMultiplier::get);
+
+  m.def("get_complex", []() { return c10::complex<double>(1.0, 2.0); });
+  m.def("get_device", []() { return at::device_of(random_tensor()).value(); });
+  m.def("get_generator", []() { return at::detail::getDefaultCPUGenerator(); });
+  m.def("get_intarrayref", []() { return at::IntArrayRef({1, 2, 3}); });
+  m.def("get_memory_format", []() { return c10::get_contiguous_memory_format(); });
+  m.def("get_storage", []() { return random_tensor().storage(); });
+  m.def("get_symfloat", []() { return c10::SymFloat(1.0); });
+  // m.def("get_symfloat_symbolic", []() { });
```

Comment:
Help! Is there a way to return a `SymFloat` in python the way I did below for `SymInt`? Is that a meaningful thing to do?

This is intentionally causing one of the new tests to fail.

Reply:
I think we forgot to write the type caster for it. It should proceed similar to how the SymInt type caster is implemented.

Reply:
I thought it was done here (although there is a TODO): https://github.com/AngelJA/pytorch/blob/95ab50dd3fcff02e092fc8a1e626cc2cf0a644f5/torch/csrc/utils/pybind.cpp#L72

I just couldn't trigger the `is_symbolic` branch without segfaulting or something like that.